frontend/src/scenes/organizationLogic.tsx

```diff
@@ -108,15 +109,13 @@ export const organizationLogic = kea<organizationLogicType>([
         deleteOrganization: async ({ organization }) => {
             try {
                 await api.delete(`api/organizations/${organization.id}`)
+                router.actions.push(router.values.currentLocation.pathname, 'organizationDeleted=true')
```

Comment:
FlyBy: Is there any reason why need to reload here? Assuming it's to clear any cache in the kea logics. If that wasn't the case, we could just push to `urls.default()` right here.